src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -2578,6 +2578,11 @@ void ApplicationManagerImpl::UnregisterApplication(
 
       return;
     }
+    if (is_resuming) {
+      resume_ctrl_.SaveApplication(app_to_remove);
```

Comment:
@AGaliuzov App is alive, but in some cases DCHECK() happens and gives core dump in processing function calls which are trying to get this App but set doesn't contain it already. For example - function is_media_app(). So resumption should be called when App exists in the set.